network/struct.go

```diff
@@ -17,80 +17,80 @@ import (
 	"golang.org/x/net/context"
 )
 
-// MaxRetryConnect defines how many times should we try to connect
+// MaxRetryConnect defines how many times we should try to connect.
 const MaxRetryConnect = 10
 
-// MaxIdentityExchange is the maximum time waited for an exchange of
-// the identity to happen.
+// MaxIdentityExchange is the timeout for an identityExchange.
 const MaxIdentityExchange = 5 * time.Second
```

Comment:
Looks like this isn't used (but maybe in one of the following PRs which depend on this one)